syllabus.md

```diff
@@ -63,13 +71,33 @@ Students who are interested in knowing how to map percentage grades onto letter
 | C-           | 70 <= x < 73 |
 | Lower Grades | x < 70       |
 
+## Participation Policy
+
+Participation grades are assigned on participation in live session. It is expected that you will attend the entirety of your live sesson and regularly participate in class discussion. These grades are assigned on a pass/fail basis. If you attend less than 50% of live sessions (don't show up at all to the session or drop off early) you will recieve a zero for your participation grade. Otherwise, you will get full credit for these points.
```

Comment:
I agree with alex that we don't want such a narrow / mechanical definition of participation.  I also would rather not overexplain what it means to participate.  Here's my attempt at a revision:
```suggestion
Participation is more than just showing up to class.  It includes being prepared, engaging actively, and treating others with respect to sustain a thriving learning environment.
```